### DISCUSSION TITLE
Use --only-match with other info

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [BurntSushi/ripgrep#2223] [Q&A] [answered]
```

❯ rga "[A-Z]{2,}" -H     
Page 82: • switched LANS, VLANs
Page 82: • virtualized networks as a link layer: MPLS
Page 82: Link Layer and LANs 6-82
...
```
\+
```
❯ rga "[A-Z]{2,}" -H -o   
VLAN
MPLS
LAN
```
=
what i want :
```
❯ rga "[A-Z]{2,}" -H -o   
Page 82: VLAN
Page 82: MPLS
Page 82: LAN
```
How can i do this in rg?
Thanks!

**Comment by BurntSushi** [comment, marked as answer]
You can't, because you're using `rga`, not `rg`. This is the wrong place for your question.

**Reply by LamprosPitsillos** [reply]
From the rga repo "rga wraps the awesome ripgrep". I was under the assumption that anything related to the arguments and interfacing was identical to ripgrep, that's why i was reading the man page of `rg` for help. 
Sorry if i was wrong, i will ask on the rga repo instead.

**Reply by BurntSushi** [reply]
Yes, but you're asking for output that's controlled by `rga` (the page numbers). ripgrep doesn't know anything about page numbers. The only numbers it knows about are line numbers, column numbers and byte offsets.
